wallet/node/Main.hs

```diff
@@ -53,6 +54,7 @@ actionWithWallet sscParams nodeParams wArgs@WalletArgs {..} =
                     (mainAction nr)
   where
     mainAction = runNodeWithInit $ do
+        whenJust walletDumpPending dumpPendingTxsSummary
```

Comment:
Originally shipped [here](https://github.com/input-output-hk/cardano-sl/pull/2213), but we decided not to backport.